javascript_nodejs/14.nlp-with-dispatch/README.MD

```diff
@@ -0,0 +1,127 @@
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
+
+This sample shows how to create a bot that relies on multiple [LUIS.ai](http://luis.ai) and [QnAMaker.ai](http://qnamaker.ai) models for natural language processing (NLP). 
+
+This bot example uses [`restify`](https://www.npmjs.com/package/restify) and [`dotenv`](https://www.npmjs.com/package/dotenv).
+
+[Dispatch](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Dispatch) is a tool to create and evaluate LUIS models used for NLP (Natural Language Processing). Dispatch works across multiple bot modules such as LUIS applications, QnA knowledge bases and other NLP sources (added to dispatch as a file type).
+
+Use the Dispatch model in cases when:
+- Your bot consists of multiple modules and you need assistance in routing user's utterances to these modules and evaluate the bot integration.
+- Evaluate quality of intents classification of a single LUIS model.
+- Create a text classification model from text files.
+
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-js.git
+    ```
+- In a terminal, navigate to javascript_nodejs/14.nlp-with-dispatch
+    ```bash
+    cd javascript_nodejs/14.nlp-with-dispatch
+    ```
+- Point to the MyGet feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules 
+    ```bash
+    npm i 
+    ```
+- Install required tools - to successfully setup and configure all services this bot depend on, you need to install the MSBOT, LUIS, QnAMaker, Ludown, Dispatch CLI tools. 
+    ```bash
+    npm i -g msbot luis-apis qnamaker ludown botdispatch
+    ```
+- Configure required LUIS, QnA Maker and Dispatch services. See [here](#configure-services)
+- Once services are configured, run the sample 
+    ```bash
+    npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+# Configure services
+This sample relies on [LUIS](http://luis.ai), [QnA Maker](http://qnamaker.ai) and [Dispatch](http://github.com/microsoft/botbuilder-tools//tree/master/packages/Dispatch) services. 
+
+## Configure the LUIS service
+To create required LUIS applications for this sample bot, 
+- Create an account with [LUIS](https://luis.ai). If you already have an account, login to your account.
```

Comment:
www.luis.ai

Reply:
To stay consistent within this doc, i've left this as https://luis.ai without the www.